exercises/hardware_access_control/README.md

```diff
@@ -64,6 +66,8 @@ Take a moment to look at the sources for these compartments in [`exercises/hardw
 If you now run the `hardware_access_part_2` firmware on the FPGA, you'll notice only `blinky_dynamic` is toggling it's LED.
 Looking at the [UART console from the FPGA][running on fpga], the following message will pop up.
 
+*Note: this only gets printed once when executing so you may have to reset your FPGA after connecting `picocom` using the reset button (SW5).*
+
```

Comment:
Again, I feel these may impede the flow and make it a little noisy. These little reminders could be put in a small section at the beginning of the exercise.

Reply:
Ok, I've removed this because it is in the running software guide.